README.md

```diff
@@ -125,8 +118,13 @@ usable as one. However, code folding and other features have impact on built-in
 
 ## Named Sections
 
-Many features rely on so-called named sections in the code.
-You define named sections with service comments like this:
+Flutter Code Editor supports `named sections` in the source code. `Named sections` are defined in the souce code by adding a tag that Flutter Code Editor recognizes.
```

Comment:
Backticks create monospace font and are traditionally used for identifiers and not the emphasis. *italic* is used for emphasis.